example/nodejs-example-realm.json

```diff
@@ -47,6 +47,8 @@
             "enabled": true,
             "publicClient": true,
             "baseUrl": "/",
```

Comment:
Not sure if there should be two `baseUrl` elements for the same client. It's working like that but I'd remove this one.